zerver/tests/test_signup.py

```diff
@@ -1664,12 +1664,13 @@ def test_successful_get_open_invitations(self) -> None:
 
         result = self.client_get("/json/invites")
         self.assertEqual(result.status_code, 200)
-        self.assert_in_success_response(
-            ["TestOne@zulip.com", hamlet.email],
-            result)
-        self.assert_not_in_success_response(
-            ["TestTwo@zulip.com", "TestThree@zulip.com", "othello@zulip.com", othello.email],
-            result)
+        invites = ujson.loads(result.content)["invites"]
+        self.assertEqual(len(invites), 2)
+
+        self.assertFalse(invites[0]["is_multiuse"])
+        self.assertEqual(invites[0]["email"], "TestOne@zulip.com")
+        self.assertTrue(invites[1]["is_multiuse"])
+        self.assertEqual(invites[1]["referrer_id"], hamlet.id)
 
```

Comment:
Have changes the tests here, because `self.assert_not_in_success_response` can give error if invite id is same as that of the user id that we do not want in the result. So, I have changed tests to check the number of invites and the email for singler user invite case and referrer id for multiuse invite.